src/components/Gradebook/footer.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import {Icon} from '@edx/paragon';
+
+export default class Footer extends React.Component {
+
+  render() {
+    return (
+      <footer role="contentinfo" aria-label="Page Footer"
+              className="d-flex justify-content-center border-top py-3 px-4">
+        <div className="max-width-1180 d-grid">
+          <div className="area-1"><img src="assets/logo-footer.png" /></div>
+          <div className="area-2">
+            <h2>edx</h2>
+            <ul className="list-unstyled">
+              <li><a href="https://www.edx.org/about-us">About</a></li>
+              <li><a href="https://www.edx.org/enterprise">edX for Business</a></li>
+              <li><a href="https://www.edx.org/affiliate-program">Affiliates</a></li>
+              <li><a href="http://open.edx.org">Open edX</a></li>
+              <li><a href="https://www.edx.org/careers">Careers</a></li>
+              <li><a href="https://www.edx.org/news-announcements">News</a></li>
+            </ul>
+          </div>
+          <div className="area-3">
+            <h2>Legal</h2>
+            <ul className="list-unstyled">
+              <li><a href="https://www.edx.org/edx-terms-service">Terms of Service &amp; Honor Code</a></li>
+              <li><a href="https://www.edx.org/edx-privacy-policy">Privacy Policy</a></li>
+              <li><a href="https://www.edx.org/accessibility">Accessibility Policy</a></li>
+              <li><a href="https://www.edx.org/trademarks">Trademark Policy</a></li>
+              <li><a href="https://www.edx.org/sitemap">Sitemap</a></li>
+            </ul>
+          </div>
+          <div className="area-4">
+            <h2>Connect</h2>
```

Comment:
Styles should be applied to classes instead of elements.

Reply:
See similar comment above.